src/frontend/.devcontainer/devcontainer.json

```diff
@@ -1,39 +1,39 @@
-// See https://containers.dev/implementors/json_reference/ for configuration reference

```

Comment:
Could you add this file/folder into the prettier ignore, wouldn't that help

Reply:
It has nothing to do with prettier, its because the line endings have changed from `CRLF` (windows) to `LF` (unix)